frontend/src/Pages/ApplicantApplicationOverviewPage/ApplicantApplicationOverviewPage.tsx

```diff
@@ -17,31 +17,15 @@ export function ApplicantApplicationOverviewPage() {
   const [admissions, setAdmissions] = useState<RecruitmentAdmissionDto[]>([]);
   const { t } = useTranslation();
 
-  function handleChangePriority(id: number, direction: 'up' | 'down') {
-    const newAdmissions = [
-      ...admissions.sort(function (a1, a2) {
-        return a1.applicant_priority - a2.applicant_priority;
-      }),
-    ];
-    const index = newAdmissions.findIndex((admission) => admission.id === id);
-    const directionIncrement = direction === 'up' ? -1 : 1;
-    if (index == 0 && direction === 'up') return;
-    if (index === newAdmissions.length - 1 && direction === 'down') return;
-
-    const old_priority = newAdmissions[index].applicant_priority;
-    const new_priority = newAdmissions[index + directionIncrement].applicant_priority;
-
-    newAdmissions[index].applicant_priority = new_priority;
-    newAdmissions[index + directionIncrement].applicant_priority = old_priority;
-
-    // TODO: Make this a single API call
-    putRecruitmentAdmission(newAdmissions[index]);
-    putRecruitmentAdmission(newAdmissions[index + directionIncrement]).then(() => {
-      setAdmissions(newAdmissions);
-    });
+  function handleChangePriority(id: string, direction: 'up' | 'down') {
+    putRecruitmentPriorityForUser(id, { direction: direction === 'up' ? 1 : -1 } as UserPriorityDto).then(
```

Comment:
Unngå casting for alt det er verdt.

```suggestion
    const data: UserPriorityDto = {...}
    putRecruitmentPriorityForUser(id, data).then(
```

Reply:
åja gud det der var røfft å se på